libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/utils/RouteUtilsTest.java

```diff
@@ -112,6 +113,24 @@ public void isArrivalEvent_returnsFalseWhenManeuverTypeIsNotArrival_andIsNotVali
     assertFalse(isArrivalEvent);
   }
 
+  @Test
+  public void isValidRouteProfile_returnsTrueWithValidProfile() throws Exception {
+    String routeProfile = DirectionsCriteria.PROFILE_DRIVING_TRAFFIC;
+
+    boolean isValidProfile = RouteUtils.isValidRouteProfile(routeProfile);
+
+    assertTrue(isValidProfile);
+  }
+
```

Comment:
It'd be great to add tests around the other valid profiles (i.e. `PROFILE_DRIVING`, `PROFILE_CYCLING` and `PROFILE_WALKING`).